addons/l10n_in_ewaybill_stock/report/ewaybill_report_views.xml

```diff
@@ -18,7 +18,7 @@
         </record>
 
         <record id="action_report_ewaybill" model="ir.actions.report">
-            <field name="name">Ewaybill</field>
+            <field name="name">Print</field>
```

Comment:
Why?

Reply:
@ajjo-odoo suggested me to rename it, as we have to print Ewaybill or Delivery challan

Reply:
Then name it like Ewaybill / Delivery challan
like https://github.com/odoo/odoo/blob/17.0/addons/sale/report/ir_actions_report.xml#L5